nautobot_golden_config/views.py

```diff
@@ -564,12 +592,31 @@ def alter_queryset(self, request):
 
     def get_extra_context(self, request, instance=None):
         """A ConfigPlan helper function to warn if the Job is not enabled to run."""
+        context = super().get_extra_context(request, instance)
+        if self.action == "retrieve":
+            context.update({"text": get_config_postprocessing(instance, request)})
```

Comment:
```suggestion
            context["text"] = get_config_postprocessing(instance, request)
```

just to be consistent